src/main/java/io/vincenzopalazzo/lightning/rest/services/PaymentService.java

```diff
@@ -165,13 +164,20 @@ public static void delInvoice(Context context) {
             content = {@OpenApiContent(from = CLightningInvoice.class)})
       }) // TODO complete the documentation
   public static void invoice(Context context) {
-    String milliSatoshi = context.formParam("msat", String.class).check(i -> !i.isEmpty()).get();
-    String label = context.formParam("label", String.class).check(i -> !i.isEmpty()).get();
+    String milliSatoshi =
+        context
+            .formParamAsClass("msat", String.class)
+            .check(i -> !i.isEmpty(), "msat value is not empty")
+            .get();
+    String label =
+        context
+            .formParamAsClass("label", String.class)
+            .check(i -> !i.isEmpty(), "label value is not empty")
+            .get();
     String description =
         context
-            .formParam("description", String.class)
-            .check(Objects::nonNull)
-            .check(i -> !i.isEmpty())
+            .formParamAsClass("description", String.class)
+            .check(i -> !i.isEmpty(), "description is not empty")
```

Comment:
I'm not able to catch what the change is here, it is only formatted right?

Reply:
This is my fault, sorry

Reply:
@vincenzopalazzo - this is now the updated way of invoking the `check` method. one has to pass a predicate and validation error. https://javalin.io/documentation#validation. prior to `4.x`, the `check` method did not require a validation error. https://javalin.io/archive/docs/v3.13.X.html#validation

Reply:
Yep thanks @strawhatasif